js/db_query.js

```diff
@@ -0,0 +1,19 @@
+const { Client } = require('pg')
+
+const func = (data, callback) => {
+  const client = new Client({
+    connectionString: data.url,
+  })
+  client.connect()
+
+  client.query(data.query, (err, res) => {
+    data.err = err
+    data.res = res
+    callback(data);
+    client.end()
+  })
+};
+
+module.exports = (data, callback) => {
```

Comment:
`callback` is outdated it does not exist anymore, use Promise approach instead, the same way like in the `http_request.js`.

Reply:
fixed